com.github.alexkdeveloper.radio.json

```diff
@@ -0,0 +1,42 @@
+{
+    "app-id" : "com.github.alexkdeveloper.radio",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "radio",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=home"
```

Comment:
why does it need home access?

Reply:
The program saves stations in the file system.

Reply:
where? 

if it uses standard location for settings, then this is not needed.

Reply:
The application creates a hidden folder in the user's home directory. That's how it does it:

```
directory_path = Environment.get_home_dir()+"/.stations_for_radio_app";
   GLib.File file = GLib.File.new_for_path(directory_path);
   if(!file.query_exists()){
     try{
        file.make_directory();
     }catch(Error e){
        stderr.printf ("Error: %s\n", e.message);
     }
     create_default_stations();
   }
   show_stations();
```

Reply:
I changed appdata.xml.in. Added the <name>tag. I changed the commit in the json file.

Reply:
The application should conform to the xdg spec instead. In the mean time you can just use `--persist=.stations_for_radio_app` instead for granting home access.

Reply:
I tried it with persist=. stations_for_radio_app. No stations are created at startup. At the first start, the program should show a list of 10 stations. The list is empty. But you can create new stations and delete them manually. Is there any way to explicitly specify the folder with stations? For example, --filesystem=/. stations_for_radio_app or something similar?

Reply:
Maybe you can leave the permissions --filesystem=home ?

Reply:
My application works with this permission:
 --filesystem=home/. stations_for_radio_app 
Can I use it?